bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "api-designer",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
We should stick to 0.2.0, since Christian wants to release with this very particular release number.